trestle/core/control_io.py

```diff
@@ -385,6 +385,24 @@ def _prompt_required_sections(self, required_sections: List[str], added_sections
             self._md_file.new_header(2, f'Control {section_title}')
             self._md_file.new_line(f'{const.PROFILE_ADD_REQUIRED_SECTION_FOR_CONTROL_TEXT}: {section_title}')
 
+    @staticmethod
+    def is_withdrawn(control: cat.Control) -> bool:
+        """
+        Determine if control is marked Withdrawn.
+
+        Args:
+            control: The control that may be marked withdrawn.
+
+        Returns:
+            True if marked withdrawn, false otherwise.
+
+        This is determined by property with name 'status' with value 'Withdrawn'.
+        """
+        for prop in as_list(control.props):
+            if prop.name == 'status' and prop.value == 'Withdrawn':
```

Comment:
I think it is better to be case-insensitive here?

Reply:
good idea.  done.